WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -543,6 +546,48 @@ + (NSString *)timeZone
   }
   return FBResponseWithOK();
 }
+
+#if __clang_major__ >= 15
```

Comment:
Or add `- (void)typeKey:(id)arg1 modifierFlags:(unsigned long long)arg2;` in https://github.com/appium/WebDriverAgent/blob/master/PrivateHeaders/XCTest/XCUIElement.h and check with selector?

On my local Xcode 14 only addressed this method, `typeKey`, but others such as `XCUIKeyboardKeyDelete` existed

Reply:
I  want to focus on current OS support as Apple updates its stuff very fast.
Although feel free to test it locally and create a PR if needed to add the support of older versions.

Reply:
Ok, i'm interested in making wda package work for both iOS 17 and 16  real devices with iOS 17's way (built with Xcode 15), so no embedded testing frameworks from `.app`. I guessed at least a selector might be needed for iOS 16 to not crash the process. I'll check for it, then create a pr to care about the case

Reply:
Just chiming in here, I'm interested in IOS 16 as well, as there currently isn't any "nice" way of mounting developer image on IOS 17 on Linux.

Otherwise, I understand wanting to support the latest, with the fast movement.